make/targets/openshift/imagebuilder.mk

```diff
@@ -0,0 +1,30 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+
+IMAGEBUILDER_VERSION ?=1.2.1
+
+IMAGEBUILDER ?= $(shell which imagebuilder 2>/dev/null)
+ifneq "" "$(IMAGEBUILDER)"
+_imagebuilder_installed_version = $(shell $(IMAGEBUILDER) --version)
+endif
+
+# NOTE: We would like to
+#     go get github.com/openshift/imagebuilder/cmd/imagebuilder@v$(IMAGEBUILDER_VERSION)
+# ...but `go get` is too unreliable. So instead we use this to make the
+# "you don't have imagebuilder" error useful.
+ensure-imagebuilder:
```

Comment:
This is not being used, but should be added right after https://github.com/openshift/build-machinery-go/blob/master/make/targets/openshift/images.mk#L12 to ensure this is being invoked before calling image target, right?

Reply:
Hah! Yes, that was the whole point of doing this :)

Reply:
Done.